examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>Does pycairo come as part of our mapper magic or is it an extra dependency?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>It does not (and installing on windows is particularly painful), so this has to be relegated to a suggestion.</p>

Reply:
<p>I've now placed this in the markdown cell above.</p>